thunder/executors/torch_compile.py

```diff
@@ -207,6 +214,12 @@ def register_supported(id: Hashable):
     prims.slice_prim,
     prims.transpose,
 }
-
 for op in supported_ops:
-    register_supported(op)
+    torch_compile_executor.register_supported(op, checker=_always_executable)
+
+
+torch_compile_complete_executor = TorchCompileExecutor(name="torchcompile_complete")
+register_executor(torch_compile_complete_executor)
+from thunder.executors.torchex import ex as pytorch_executor
+
+torch_compile_complete_executor._implmap = dict(pytorch_executor.implmap)
```

Comment:
Let me know if there's a better way to do this. I'm also not sure if I should copy the lookasides too